### PR TITLE
Fix issue with "post_peek_screenshot" action failing when it's not supposed to

### PR DESCRIPTION
**Pull request by Thomas-Boi**
This PR should fix the issue mentioned in #514 .

What's new:
The "post_peek_screenshot" will now check for three different states of the trigger "peek_icons" action: "skipped", "success", and "failure. These values have to be found by console logging the `github.event` object. The docs don't mention this for some reason.

image

Also, GitHub Actions will construct env variables before an if check by default:
image

Before, there was an env variable that relies on a skipped step => script was failing since it couldn't construct this variable.

That's pretty much it. I've tested it using my forked branch. The behaviours are:
- If "peek_icons" was skipped (label incorrect), the "post_peek_action" would also skip
- If "peek_icons" was successful, the "post_peek_action" would post successful message
- If "peek_icons" failed (either due to user or system error), the "post_peek_action" would post failure message. It'll be up to us to find what's wrong. 

I thought about separating user and system error but I don't think it's worth it.  It'll require some extra code and the benefit (in my opinion) is not worth it. If I was to do it, it'll be something like the check SVG workflow.

Let me know what you think

### CODE REVIEW

File: .github/workflows/peek_icons.yml
Comment (Panquesito7) [inline .github/workflows/peek_icons.yml:26]:
```suggestion
      - name: Save the PR number in an artifact
```

File: .github/workflows/peek_icons.yml
Comment (Panquesito7) [inline .github/workflows/peek_icons.yml:32]:
```suggestion
      - name: Upload the PR number
```

File: .github/workflows/peek_icons.yml
Comment (Panquesito7) [inline .github/workflows/peek_icons.yml:33]: I think it's better to use the `master` version here. It's more updated than the `v2` branch.
```suggestion
        uses: actions/upload-artifact@master
```

**Comment by Panquesito7** [comment]
Weird, not sure why here it displays two `Peek Icons`. 🤷‍♂️

image

At the bottom, it only displays one. 🤔

File: .github/workflows/peek_icons.yml
Comment (Thomas-Boi) [reply .github/workflows/peek_icons.yml:33]: I think it might be better to stay with v2 or at least a specific tag. That way, if the upload-artifact repo is updated and breaking change occurs, we don't have to worry about it until we upgrade the action.

**Comment by Thomas-Boi** [comment]
> Weird, not sure why here it displays two `Peek Icons`. 🤷‍♂️
> 
> At the bottom of the PR, it only displays one. 🤔

Oh, I know this one. Currently, our peek action is triggered by the "bot:peek" label. To do this, we create an event whenever a label is added and check the name:

image

If the name doesn't match, it skips. 

Currently, we have 2 labels. Since they aren't "bot:peek", the action will be skipped, thus the above situation.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you! 🎉 👍 
Let's wait on @amacado's approval. 🙂

File: .github/workflows/peek_icons.yml
Comment (amacado) [reply .github/workflows/peek_icons.yml:33]: @Thomas-Boi is right, (almost) always stick to a specific version and do not target a `master` or `latest` branch. Targeting a `latest`-release is not secure in terms of code injection attacks. You might wanna take a look at https://github.com/devicons/devicon/issues/395.

**Review by amacado — APPROVED** [review]
Good explanation @Thomas-Boi!

**Pull request merged**
